### PR TITLE
adding FIXX-074526 token branding

### PR DESCRIPTION
**Pull request by ctimaru**
Adding branding for FIXX token

### CODE REVIEW

**Comment by ctimaru** [comment]
@SeverMM Any other requirements? Please let us know.

**Comment by ctimaru** [comment]
@SeverMM @sasurobert Can you please review PR #237 for branding. We had an allocation and the tokens are circulating. Would like to avoid scams. Ty.

**Comment by LucianMincu** [comment]
Hi there! 

Would it be possible to update your fork with the latest updates from the main repo? 

Thanks!

**Comment by ctimaru** [comment]
> Hi there!
> 
> Would it be possible to update your fork with the latest updates from the main repo?
> 
> Thanks!

Hi Lucian! 

Done. Thank you!

**Review by AlexandruPislariu — APPROVED** [review]

File: tokens/FIXX-074526/info.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "website": "https://fixxfi.online/",
+    "description": "Token utility of FixxFi. Rapairs made easy by installments",
+    "social": {
+        "email": "blockchain@fixxfi.com",
+        "blog": "https://fixxfi.medium.com/",
+        "twitter": "https://twitter.com/FixxToken",
+        "telegram": "https://t.me/fixxtoken",
+        "whitepaper": "https://fixxfi.medium.com/fixxfi-white-paper-25f67384ea09",
```
Comment (AlexandruPislariu) [inline tokens/FIXX-074526/info.json:9]: Hello!
The comma on line 9 should be removed.
Please make sure to validate JSON with [https://jsonlint.com/](https://jsonlint.com/) before opening the PR.

**Review by AlexandruPislariu — CHANGES_REQUESTED** [review]

File: tokens/FIXX-074526/info.json (hunk 1 above)
Comment (ctimaru) [reply tokens/FIXX-074526/info.json]: > Hello! The comma on line 9 should be removed. Please make sure to validate JSON with https://jsonlint.com/ before opening the PR.

Hello! 
Removed and validated the JSON. 
Thank you!

**Review by AlexandruPislariu — APPROVED** [review]

**Comment by ctimaru** [comment]
Hi @LucianMincu, 
If everything looks good, can we have it merged? 
Thank you!

**Comment by LucianMincu** [comment]
Done!

**Pull request merged**